202305_coastal_ecosystems_TC/impact_calculation/Impact_calculation.py

```diff
@@ -0,0 +1,33 @@
+"""
+Script to create calculate impact for given population exposure
+Author: Sarah Hülsen
+"""
+
+from climada.entity import Exposures
+from climada.hazard import Hazard
+from modules.CatImpactFuncTC import cat_impf_set
+from modules.CatImpactTC import imp_per_cat
+
+
+basin = 'AP'    # TC basin (e.g. AP, IO, SH, WP)
+year = '2020'   # population exposure year (e.g. 2000 or 2020)
+scenario = 'hist_STORM'  # current climate or SSP585 TC hazard (hist_STORM or clim_STORM_{model})
+
+# Data paths
+input_path = '../results/intermediate/'
+output_path = '../results/final/'
```

Comment:
```suggestion
from pathlib import Path

from climada.entity import Exposures
from climada.hazard import Hazard
from modules.CatImpactFuncTC import cat_impf_set
from modules.CatImpactTC import imp_per_cat


basin = 'AP'    # TC basin (e.g. AP, IO, SH, WP)
year = '2020'   # population exposure year (e.g. 2000 or 2020)
scenario = 'hist_STORM'  # current climate or SSP585 TC hazard (hist_STORM or clim_STORM_{model})

# Data paths
input_path = Path('../results/intermediate/')
output_path = Path('../results/final/')
```

It is convenient to use the `pathlib` library as this takes care of the different paths between operating systems. Your path here would not work on windows for example.

I suggest you edit all path definitions using this library.